### PR TITLE
Meson: new version 0.61.1

### PR DESCRIPTION
**Pull request by kjrstory**
Added Meson version 0.61.1 due to its requirement in "su2" package installation.
Please refer this commit of SU2
https://github.com/su2code/SU2/commit/fc37e0067f53c0eb712306d07ea3339360113997

### CODE REVIEW

**Review by eli-schwartz — CHANGES_REQUESTED** [review]
I disagree with this change because it is an old and buggy version of meson that has later patch/bugfix releases within the same minor release series, which spack already packages.

Furthermore, SU2 no longer needs this due to https://github.com/su2code/SU2/pull/1951

**Comment by kjrstory** [comment]
I agree with @eli-schwartz opinion.
However, currently, this commit(https://github.com/su2code/SU2/pull/1951) has not been applied to any released versions of SU2.
As I have tested, the restriction was implemented in a specific range of versions (@7.4.0:7.5.1) of SU2, but I have successfully installed SU2 with a different version of meson(1.1.0).
In light of this, I have created the patch and submitted a pull request (#37767). 
I would greatly appreciate your feedback on the meson version included in this PR.

**Comment by adamjstewart** [comment]
I'll let the maintainers decide on this one. I personally don't think it's a big deal to add an old version, it won't get used by default other than by this package.

**Comment by eli-schwartz** [comment]
I'd be less concerned about adding an old version if it were the latest bugfix release of that old version. :D

Spack doesn't currently package the .5 bugfix release for this old minor release. I'd have no objections to adding that -- it fixes bugs that may be encountered by people who have made the judgment call that their needs are best served by sticking to that old version and avoiding sweeping changes, deprecations, and whatnot, from newer versions.

As an upstream meson maintainer, I want to push people to always upgrade to the latest bugfix release, if they aren't in a position to upgrade to the latest feature release. That's why we have 0.61.x as an LTS in the first place.

But for software that locks itself to an exact bugfix release, the best solution is to backport an upstream change that removes the locked dependency. That's what the other PR does, so this PR won't be needed even for SU2 anymore. :)

**Pull request closed without merging**
